recommender/commands/train.py

```diff
@@ -44,7 +44,7 @@ def _rl_v2():
 def _all():
     """Run all training pipelines"""
     AEPipeline(AUTOENCODERS_PIPELINE_CONFIG)()
-    EmbeddingComponent()()
+    EmbeddingComponent()(verbose=True)
```

Comment:
we shloud delegate choice of being verbose or not to the user (developer) calling our cli - e.g add verbose option to click

Reply:
I agree but also I think that it should be the scope for a new ticket - make it easy to enable or disable being verbose for every command.

Reply:
I made a ticket https://github.com/cyfronet-fid/recommender-system/issues/360